diff --git a/config.def.h b/config.def.h
index 9855e21..1c3cc85 100644
--- a/config.def.h
+++ b/config.def.h
@@ -1,12 +1,14 @@
+#include <luca/preferences.h>
+
 /* user and group to drop privileges to */
 static const char *user  = "nobody";
 static const char *group = "nogroup";
 
 static const char *colorname[NUMCOLS] = {
-	[INIT] =   "black",     /* after initialization */
-	[INPUT] =  "#005577",   /* during input */
-	[FAILED] = "#CC3333",   /* wrong password */
+	[INIT] =   background,     /* after initialization */
+	[INPUT] =  blue,					 /* during input */
+	[FAILED] = red,            /* wrong password */
 };
 
 /* treat a cleared input like a wrong password (color) */
-static const int failonclear = 1;
+static const int failonclear = 0;
